### ISSUE TITLE
Is master broken?

### ISSUE THREAD

**Issue by claudiopro** [workshopper/workshopper-adventure#16]
Hi @martinheidegger, is master broken? I ran the example `foo` adventure like this:

```
git clone git@github.com:workshopper/workshopper-adventure.git
cd workshopper-adventure
npm install
examples/foo.js
<nothing>
```

**Comment by claudiopro** [comment]
The package version published on `npm` (v4.2.1) is also out of sync with `package.json` on master (v4.1.0), am I missing something? :confused:

**Comment by martinheidegger** [comment]
It might be that you just accessed the repo while I was publishing the the code: The latest version is 4.2.2

Aside from that you are talking about a larger issue: The current documentation is out-of-sync and the current examples are not very good. If you look at `learnyounode`: It exports in [learnyounode.js](https://github.com/workshopper/learnyounode/blob/cefeae556ed72ca178b4bea03df4242dface6a6f/learnyounode.js#L28) and then has a separate file [bin/learnyounode](https://github.com/workshopper/learnyounode/blob/cefeae556ed72ca178b4bea03df4242dface6a6f/bin/learnyounode) for the binary that is reference respectively in the `package.json` [main](https://github.com/workshopper/learnyounode/blob/cefeae556ed72ca178b4bea03df4242dface6a6f/package.json#L56) and [bin](https://github.com/workshopper/learnyounode/blob/cefeae556ed72ca178b4bea03df4242dface6a6f/package.json#L54).

**Comment by claudiopro** [comment]
Ah! Gotcha. How can I help with documentation? From PR #17 I see the fix for v4.2.x is calling `execute()` on the return value of `workshopper()` i.e. the function exported by the `workshopper-adventure` module, is that right?

**Comment by martinheidegger** [comment]
I have started to add issues to this repository with the labels "documentation" and "automization". maybe you could look at them?
